delta/sdk/src/main/scala/ch/epfl/bluebrain/nexus/delta/sdk/model/ResourceUris.scala

```diff
@@ -55,7 +55,17 @@ object ResourceUris {
     def incomingShortForm(implicit base: BaseUri): Uri = accessUriShortForm / "incoming"
     def outgoingShortForm(implicit base: BaseUri): Uri = accessUriShortForm / "outgoing"
     def project(implicit base: BaseUri): Uri           = ResourceUris.project(projectRef).accessUri
+  }
 
+  /**
+    * A resource that is rooted in a project but not persisted or indexed.
+    */
+  final case class EphemeralResourceInProjectUris(
```

Comment:
I understand that's just to get rid of outgoing and incoming links, but I don't know if its worth it. The incoming and outgoing endpoints will just work and return an empty result, which is fine.

Besides notice that incoming links might be present on the platform, from some people that created a resource and linked it to the id of the archive.

Basically, you can keep it or remove it, as you like :)

Reply:
I would keep it as is, since the example is an edge-case that most likely will never happen.